### PR TITLE
feat: add event page

### PR DESCRIPTION
**Pull request by vatsalsinghkv**
### Describe the changes you've made

Added event page for the upcoming events

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/UniKonf/vibey/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Deepu178** [comment]
Share the screenshot @vatsalsinghkv

**Review by Deepu178 — APPROVED** [review]

**Review by money8203 — APPROVED** [review]

**Comment by vatsalsinghkv** [comment]
# Screenshot

Screenshot 2023-04-09 at 12 12 52 AM
 
@Deepu178 @money8203

File: next.config.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ const nextConfig = {
   eslint: {
     dirs: ['src'],
   },
+  images: {
+    domains: ['source.unsplash.com', 'hackcbs.tech'],
```
Comment (Chandraprakash-Darji) [inline next.config.js]: download this images and put in assets,

File: src/components/upcoming/Events.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { EventType } from '@/lib/types';
+
+import Event from '@/components/upcoming/Event';
+
+import React from 'react';
+
+type Props = { events: EventType[] };
+
+const Events = ({ events }: Props) => {
```
Comment (Chandraprakash-Darji) [inline src/components/upcoming/Events.tsx]: Remove this component, just directly include in the page, I think @Deepu178  what do think?

File: src/pages/events/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
+    link: '#',
+    image: 'https://source.unsplash.com/random?300x300?hack,computers',
+  },
+  {
+    id: 'id4',
+    name: 'HackCBS',
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
```
Comment (Chandraprakash-Darji) [inline src/pages/events/index.tsx]: It will be good that you pass the time zone, reason: people from outside of india may see wrong date

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/upcoming/Events.tsx (hunk 2 above)
Comment (vatsalsinghkv) [reply src/components/upcoming/Events.tsx]: Fair enough. Will restructure it ❤️

File: src/pages/events/index.tsx (hunk 3 above)
Comment (vatsalsinghkv) [reply src/pages/events/index.tsx]: @Chandraprakash-Darji could you elaborate it? Like what'd be the recommend way to pass the timezone?

**Comment by Deepu178** [comment]
@vatsalsinghkv is this ready for the merge?

**Comment by vatsalsinghkv** [comment]
> @vatsalsinghkv is this ready for the merge? 

@Deepu178 Almost, just discussing the remaining changes (the timezone issue) with @Chandraprakash-Darji

**Review by Chandraprakash-Darji — APPROVED** [review]

**Review by Deepu178 — APPROVED** [review]

**Pull request merged**